app/assets/stylesheets/content/_simple_filters.sass

```diff
@@ -91,6 +91,24 @@ $filters--border-color: $gray !default
     @include text-shortener
     margin: auto 0
 
+fieldset#date-range
```

Comment:
Is there a way to solve this in a more generic way? so instead of styling via the ID, add a modifier class for the case of radio buttons in the filters. I am trying to get rid of such specific styles.